testing/extremetuxracer/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: Danct12 <danct12@disroot.org>
+# Maintainer: Danct12 <danct12@disroot.org>
+pkgname=extremetuxracer
+pkgver=0.7.5
+pkgrel=0
+pkgdesc="Downhill racing game starring Tux"
+url="https://sourceforge.net/projects/extremetuxracer"
+arch="all"
+license="GPL-2.0-only"
+makedepends="sfml-dev glu-dev freetype-dev"
+source="https://downloads.sourceforge.net/${pkgname}/etr-${pkgver}.tar.xz"
```

Comment:
Replace `$pkgname` for `extremetuxracer`. Also, not really important, but the brackets (`{}`) are not required if you're not modifying the variable.